### PR TITLE
Fix/smoother onboarding

### PR DESCRIPTION
**Pull request by alfietham**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules
 multichat*/
+multi-chat*/
```
Comment (sergueyarellano) [inline .gitignore:3]: agree

File: chat.js [hunk 2]
```diff
@@ -44,6 +44,22 @@ const topicSlug = slug(topic)
 const nickname = slug(argv.n)
 logLogo()
 
+// TOOD: https://github.com/SBoudrias/Inquirer.js#layouts
```
Comment (sergueyarellano) [inline chat.js:47]: pull the changes first alfie

**Review by sergueyarellano — COMMENTED** [review]
@alfietham closing this, can you reopen rebasing new changes from master?
thanks man!

**Comment by alfietham** [comment]
Duuuude. Okay this branch is not needed anymore as the fixes has been incorporated in your commits in master

**Pull request closed without merging**